src/java/htsjdk/samtools/cram/ref/CRAMReferenceSource.java

```diff
@@ -0,0 +1,21 @@
+package htsjdk.samtools.cram.ref;
+
+import htsjdk.samtools.SAMSequenceRecord;
+
+/**
+ * Interface used to supply a reference source when reading CRAM files.
+ */
+public interface CRAMReferenceSource {
+
+    /**
+     * getReferenceBases
+     * @param sequenceRecord the SAMSequenceRecord identifying the reference
+     *                       being requested
+     * @param tryNameVariants if true, attempt to match the requested sequence name
+     *                        against the reference by using common name variations,
+     *                        such as adding or removing a leading "chr" prefix
+     *                        from the requested name. if false, use exact match
+     * @return the bases representing the requested sequence
```

Comment:
Documented to return null, since that is what the current sole implementation does.